content/introduction/what-is-dolt.md

```diff
@@ -0,0 +1,46 @@
+---
+title: What is Dolt
+---
+
+## Version Controlled Database
+Dolt is a version controlled relational database. It has a SQL query interface, which draws heavily on the MySQL dialect. It implements Git-like version control features.
```

Comment:
It has a MySQL compatible SQL interface. The second sentence is Oscar. Every sentence that has a comma, go remove the comma.

Reply:
I would end this with something like, Dolt is like Git and MySQL had a baby. Or of that's too whimsical for your sensibilities. Dolt is best described as a combination of Git and MySQL.

Reply:
How about:
> Dolt implements a superset of MySQL. It is compatible with MySQL, and has additional functionality for Dolt's version control features.

Reply:
I like that.